renku/core/commands/workflow.py

```diff
@@ -274,30 +284,79 @@ def _export_workflow(
     if output:
         output = Path(output)
 
+    if values:
+        values = _safe_read_yaml(values)
+        rv = RunValues(workflow, values)
+        workflow = rv.apply()
+        if rv.missing_parameters:
+            communication.warn(
```

Comment:
i've tested this on a local instance and for some reason `communication.warn` doesn't do anything, i.e. i'm not getting getting anything on the stdout/stderr

Reply:
Did you set up the communication provider in the command builder? Like https://github.com/SwissDataScienceCenter/renku-python/blob/master/renku/cli/dataset.py#L467 and https://github.com/SwissDataScienceCenter/renku-python/blob/master/renku/cli/dataset.py#L472

Reply:
🤦 this is the result of a quickfix. thnx!